l2tdevtools/build_helpers/dpkg.py

```diff
@@ -751,7 +751,10 @@ def _DetermineBuildConfiguration(self, source_directory):
     else:
       build_configuration = dpkg_files.DPKGBuildConfiguration()
 
-      if os.path.exists(os.path.join(installroot_path, 'usr' 'bin')):
+      if os.path.exists(os.path.join(installroot_path, 'usr', 'bin')):
+        build_configuration.has_bin_directory = True
+
+      if os.path.exists(os.path.join(installroot_path, 'usr', 'local', 'bin')):
```

Comment:
remove this, instead add `--prefix=/usr` to the command defined on line 742

Reply:
for some reason, adding --prefix also means python packages end up in 
python2.7/site-packages  instead of python2.7/dist-packages

I still need to investigate

Reply:
This is apparently WAI:
https://bugs.launchpad.net/ubuntu/+source/python2.6/+bug/362570

Reply:
per conversation this is fine. this install is used to detect features to generate the dpkg packaging files, those will build dist-package. Let's put a comment here as a reminder.